nemo_skills/pipeline/megatron_lm/train.py

```diff
@@ -208,6 +213,17 @@ def train_megatron_lm(
         extra_arguments=extra_arguments,
     )
 
+    # Add extra sbatch arguments
```

Comment:
could you please move this into a new function inside nemo_skills/pipeline/utils/cluster.py (or another file in that folder), so that we can make it just a one-line change in each of the pipeline scripts and reuse all logic?